.gitignore

```diff
@@ -8,3 +8,4 @@ it
 .tmpenv
 .direnv
 .vscode
+.DS_Store
```

Comment:
Please do not make unrelated edits in PRs. You can ignore paths locally by adding them to `.git/info/exclude`. If you think this should be added to the global `.gitignore` please open a separate PR (although I feel more like removing idea and vscode looking at it, these are just dev-specific and should not be global).

Reply:
Roger. Removed.